app/src/routes/course.js

```diff
@@ -83,7 +84,7 @@ router.get('/:id', async function(req, res) {
             res.status(500).send(body)
         } else if (e instanceof AuthError) {
             res.status(401).send(body)
-        } else if (e instanceof NotFoundError) {
+        } else if (e instanceof ServerError) {
```

Comment:
este server error qué estaría representando?

Reply:
Cualquier error del back de python, no se me ocurria un mejor nombre. OSea no un problema de coneccion, sino un error del back en si.

Reply:
yo los diferenciaría..no es lo mismo un forebidden, un bad request que un not found por nombrar algunos casos que puede devolver python aca...